logic/minecraft/auth/tasks/AuthenticateTask.cpp

```diff
@@ -14,20 +14,15 @@
  * limitations under the License.
  */
 
-#include <auth/flows/AuthenticateTask.h>
```

Comment:
I don't want to see any changes made to this. It was time tested and without a single known issue.